packages/flutter_driver/lib/src/driver/gc_summarizer.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'timeline.dart';
+
+/// GC related timeline events.
+///
+/// All these events occur only on the UI thread and are non overlapping.
```

Comment:
It that really guaranteed to be true? These events occurring on the UI thread only I mean. For one, there can be multiple UI threads in a process.